### ISSUE TITLE
Full sklearn compliance

### ISSUE THREAD

**Issue by falkben** [neurodata/SPORF#258]
So that it passes the estimator test.

First thing to do is to make the model pickleable.

**Comment by adam2392** [comment]
Hi @falkben was there ever any update on pickling the forest classifiers to be ran through something like joblib?

**Comment by falkben** [comment]
No, I'm sorry, I don't have an update on that. My guess is that it won't work with joblib's `dump` method. Have you tried it?

**Comment by pepetikesavasiddhardha** [comment]
@falkben I tried to pickle it with joblib and pickle libraries but it was not working, its showing an error like this
**TypeError: cannot pickle 'pyfp.fpForest' object**
So can u suggest me how can i save this method? Is there any way to do that?

**Comment by adam2392** [comment]
@pepetikesavasiddhardha currently the tree models are implemented inline with sklearn in https://github.com/neurodata/scikit-tree
